docs/dev_guide/diagnostic_ref/dpct1106.rst

```diff
@@ -0,0 +1,57 @@
+.. _id_DPCT1106:
+
+DPCT1106
+========
+
+Message
+-------
+
+.. _msg-1106-start:
+
+``<function name>`` was migrated with the Intel extensions for device information which
+may be not supported by all compilers or runtimes. You may need to adjust the code.
+
+.. _msg-1106-end:
+
+Detailed Help
+-------------
+
+The CUDA ``<function name>`` for example ``cudaMemGetInfo`` or ``cuMemGetInfo`` was
+migrated with Intel's Extensions for Device Information, which may be not supported
+by all compilers or runtimes.
```

Comment:
"The **CUDA*** **function** ``<function name>`` (for example ``cudaMemGetInfo`` or ``cuMemGetInfo``) was
 migrated with **the** **Intel extensions for device information**, which may be not supported
 by all compilers or runtimes. **You may need to adjust the code.**"